### ISSUE TITLE
Unable to enable firewall group, without explicitly naming rules in the group.

### ISSUE THREAD

**Issue by RRohi** [ansible-collections/community.windows#132]
##### SUMMARY

`win_firewall_rule` should support enabling firewall rules by groups.

By providing **group name**, the **name** should not be required.
##### ISSUE TYPE
Incomplete functionality.

##### COMPONENT NAME

win_firewall_rule

##### ADDITIONAL INFORMATION

Problem now is that after **group** is provided, **name** is still required, which means that although I provided **Group name**, only one rule will be enabled, one that matches the **name**.

The provided task will enable Remote Event Log Management and Windows Management Instrumentation (WMI) firewall **rules**, not **groups**.

Expected usage scenario would be that I don't have to provide firewall rule names explicitly from the firewall group. Enable all the rules within the **Remote Event Log Management** group, for example.

```yaml
- name: Enable Remote Event Log Management & WMI Groups
  win_firewall_rule:
    direction: in
    name: "{{ item }}"
    group: "{{ item }}"
    enabled: yes
    profiles: domain,private
    action: allow
    protocol: tcp
  loop:
    - Remote Event Log Management
    - Windows Management Instrumentation (WMI)
```

**Comment by jborean93** [comment]
Was recently implemented with https://github.com/ansible-collections/community.windows/pull/127. Are you having problems with the new code.

**Comment by RRohi** [comment]
I don't think I know how to update community modules manually. All I have is whatever came with Ansible installation.

Tried `ansible-galaxy collection install community.windows` from project README, but the modules (community.windows:1.0.0, ansible.windows:1.0.0) got installed in the wrong place, maybe? (~/.ansible/collections/ansible_collections/*/windows)

Or maybe not, but I ended up with an error regardless, although a new one:
```vim
failed: [IP] (item=Remote Event Log Management) => {"ansible_loop_var": "item", "changed": false, "item": "Remote Event Log Management", "msg": "Get-AnsibleParam: Missing required argument: name"}
failed: [IP] (item=Windows Management Instrumentation (WMI)) => {"ansible_loop_var": "item", "changed": false, "item": "Windows Management Instrumentation (WMI)", "msg": "Get-AnsibleParam: Missing required argument: name"}
```

Used task:
```yaml
- name: Enable Remote Event Log Management & WMI Groups
  win_firewall_rule:
    direction: in
    group: "{{ item }}"
    enabled: yes
    profiles: domain,private
    action: allow
    protocol: tcp
  loop:
    - Remote Event Log Management
    - Windows Management Instrumentation (WMI)
```

**Comment by jborean93** [comment]
That error indicates that it's still using the builtin one. To use the modules in this collection you need to be on Ansible 2.9, but 2.10 is preferable and is what it has been tested with. Change your module to `community.windows.win_firewall_rule` for Ansible to force the use of the collection version. You can also set `collections: ['community.windows']` on your play like the below but I'm unsure if that will override the builtin version.

```
- hosts: windows
  collections:
  - community.windows
  tasks:
  - win_firewall_rule:
      ...
```

In any case the collection at v1.0.0 will be included in Ansible 2.10 when that is released hopefully sometime next month.

**Comment by RRohi** [comment]
I'm using 2.9.9 version.

If this functionality will become default by next month, hopefully?, then I'll just wait. Thanks for the feedback.
